programs/bpf_loader/src/syscalls/mod.rs

```diff
@@ -239,6 +239,24 @@ macro_rules! register_feature_gated_function {
     };
 }
 
+pub fn morph_into_deployment_environment_v1(
+    from: Arc<BuiltinProgram<InvokeContext>>,
+) -> Result<BuiltinProgram<InvokeContext>, Error> {
```

Comment:
We should probably do this in RBPF instead and have a `set_config` or a copy constructor there.

Reply:
Do you want to push an update to rBPF crate for this?
We also need to backport it to v1.18. So maybe leave this here for time being, and remove it from master once rBPF change is available?

Reply:
We can remove it from master in the next RBPF release.